### ISSUE TITLE
Broken app label: expressvpn

### ISSUE THREAD

**Issue by jbart7272** [Installomator/Installomator#2001]
**What's wrong with the app label?**
A clear and concise description of what the issue is. (Ex: The app label's downloadURL is coming up blank)

**Installomator version**
What version are you running? You must include the date. (Ex: Installomator v. 10.7beta, date 2024-08-31. Be specific if running the beta!)

**Additional context**

**Installomator log**
**At the bottom of this report**, provide a log, either by running Installomator in Terminal and saving the output, or by downloading the result log from your MDM. [Debug logging level](https://github.com/Installomator/Installomator/wiki/Configuration-and-Variables#logging-level) is not needed for easily identified issues, like a `downloadURL` is blank or an `appNewVersion` is incorrect. *PLEASE format the log [using a code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks)!*

**Comment by uurazzle** [comment]
FYI:

Here is a PR, https://github.com/Installomator/Installomator/pull/2941, that fixes the issue, waiting on maintainers' review, approval, and merge.

For example, here is the updated label:

```
expressvpn)
    name="ExpressVPN"
    type="pkg"
    downloadURL="https://www.expressvpn.com/clients/latest/mac"
    appNewVersion="$(curl -fsIL "https://www.expressvpn.com/clients/latest/mac" | grep -i "^location:" | sed 's/.*expressvpn_mac_//i; s/_release\.pkg.*//; s/\.[0-9]*$//')"
    expectedTeamID="TC292Y5427"
    ;;

```

**Comment by uurazzle** [comment]
Hi @jbart7272:

Thanks for the report.

### Issue fixed on main

Good news -- this has already been fixed on the `main` branch. The label(s) reported in this issue now resolve and download correctly:

- `expressvpn` -- version detected: 11.71.0

The fix was merged via:
- #2941 -- expressvpn-label-update

The updated label will be included in the next Installomator release. In the meantime, you can run from `main` by assembling locally or waiting for the next tagged release. Closing this issue -- feel free to reopen if you continue to see problems. Thanks!
